_careers/data.md

```diff
@@ -0,0 +1,140 @@
+
+We have landed the first clients and have very exciting prospects for the coming months.
+
+We value
+
+- **open**: source, data, knowledge
+- **balance**: design your work schedule around your life, not the other way round
+- **communication**: remote-first approach. Good & orderly written communication > meetings > real time interruptions
+- **quality**: pride in process and results
+- **independence**: responsibility for your own work, not micromanaging
+
+
+## The job
+
+One of Gobierto Contracts' parts is a global search engine of tenders and contracts extracted from public sources. There are some many insights that can be extracted from this data, but hardly no one is doing it. We want to put this data to work to improve public spending on different dimensions (efficiency, control, transparency...). Some of the tasks:
+
+- Data extraction: from structured sources and not structured ones (web pages, PDFs). We already have many data ingestion processes running, you'll help improve them if needed.
+- Quality control of data: we have started to build some automated checks to validate the quality of new data. You'll help consolidate and improve it
+- Data cleaning: you know the drill :)
+- Data analysis and exploratory visualizations to build, communicate and reason about hipothesis
+- Prediction and clasification: we have already built some initial models to predict the number of offers a tender will receive. We want to do many other things in this front. Predict prices, discounts, detect possible frauds...
+- Entity extraction: we have tons of PDFs with unstructured content (companies, concepts, contract clauses...). Once we have extracted them, many important products can be built.
+
+And many other challenges that will appear along the way. We'll start bit by bit and grow the team which you may end up managing (if that's something you'd want, that is).
+
+We use Jenkins to automate tasks. We are a Ruby On Rails shop, so some Ruby knowledge would be ideal, but we understand you'll be using Python and the usual suspects.
+
+You'll also be contributing from time to time to Gobierto Transparency, our open source solution for transparency and accountability initiatives.
+
+We work on [Github](http://github.com/populatetools/) using a git workflow with CI/CD, tests, different automations... Our workflow has been polished throughout a few years, so have automated many things (monitoring, error tracking, server setups & updates, deployments, various checks...). You can take a look at the [Gobierto repo to see how we actually work](http://github.com/populatetools/gobierto).
+
+
+## Day to day work
+
+- **Plan** new features of Gobierto and existing applications, and contribute to new apps from scratch
+- **Implement** defined features, manage your sprints, do sprint reviews
+- **Collaborate** with other team members: provide feedback and do code reviews
+- **Improve** and maintain existing code of production applications
```

Comment:
These are developer tasks, we could change them to:

- Plan: analyze and suggest new analysis over the data
- Implement: develop your scripts and visualizations, prepare notebooks
- Present results: present results to the team, review suggestions and iterate

Reply:
Perfect